dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -133,6 +143,28 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::Unregiste
 }
 CATCH_RETURN()
 
+STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::AddToastRegistrationMapping(_In_ PCWSTR processName, _In_ PCWSTR appId) noexcept try
```

Comment:
Why are you not calling this in CreatePushChannel anymore?

Reply:
It should be in CreatePushChannel. I added it back.